test/npm/test-npm-install.js

```diff
@@ -99,8 +99,13 @@ test('npm-install: failed install', function (t) {
       npmLevel: 'http'
     }
   };
+  var expected = {
```

Comment:
OH RAD I had no idea you could write an object with keys that are regex's... this is super cool